src/arrayMethodLastIndexOf.js

```diff
@@ -4,8 +4,16 @@
  * Implement method lastIndexOf
  */
 function applyCustomLastIndexOf() {
-  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex = 0) {
+    const arrayLength = this.length;
+    if (arrayLength === 0) return -1;
+    if (fromIndex >= this.length) return -1;
+    if (searchElement === undefined) return -1;
```

Comment:
combine into 1 if

Reply:
why? as for me its easier to read than alot of statements with or/and operator?

Reply:
@YuriiHoliuk

Reply:
done